### DISCUSSION TITLE
Toggleable Text Column Table Not Working on Stack Grid

### DISCUSSION THREAD

**Discussion by AFSDARIF** [filamentphp/filament#13024] [General]
I use toggleable in text column table.

![Dingtalk_20240531092538](https://github.com/filamentphp/filament/assets/10229660/3e2c02f5-cdc5-436d-a87c-f6ee5a34a883)

Stack::make([
Tables\Columns\TextColumn::make('DataBox')
->toggleable(isToggledHiddenByDefault: true)
->size('sm')
->wrap()
->html(),

Tables\Columns\TextColumn::make('DataPcs')
->toggleable(isToggledHiddenByDefault: true)
->size('sm')
->wrap()
->html(),
])

**Comment by arshaviras** [comment]
Same issue, did you find solution?

**Comment by pjotrvdh** [comment]
I'm also having this issue, also looking for the solution
